_sources/Unit5-Writing-Classes/topic-5-1-parts-of-class.rst

```diff
@@ -158,17 +240,29 @@ Instance Variables
    pair: class; instance variables
 
 
-**Instance Variables** hold the data for an object.  They record what an object needs to know to do work in the program.  Instance variables are also called **attributes**, **fields**, or **properties**.
+As we’ve said, **instance Variables** hold the data for an object. They record
+what an object needs to know to play its role in the program. Instance variables
+are also sometimes called **attributes**, **fields**, or **properties**.
 
-Instance variables in general and on the AP CSA exam should be declared **private**.  Think of ``private`` as like your diary.  Only you should have direct access to it.  In this case ``private`` means that only the code in this class can directly access the instance  variable values.
+In general and definitely on the AP CSA exam, instance variables should be
+declared **private**. Think of ``private`` as like your diary. Only you should
+have direct access to it. Similarly, in Java a ``private`` instance variable can
+only be accessed by code in the class that declares the variable.
 
 .. note::
 
    Instance variables are declared right after the class declaration.  They usually start with ``private`` then the *type* of the variable and then a *name* for the variable. Private means only the code in this class has access to it.
 
-The ``Person`` class declares 3 private instance variables: name, email, and phoneNumber. These are things that you might want to know about a person.  They are declared at the top of the class and they exist inside the { } of the class. The methods of the class share the instance variables. They can access and use them.
+The ``Person`` class declares 3 private instance variables: ``name``, ``email``,
+and ``phoneNumber``. These are things that you might want to know about a
+person. They are declared at the top of the class and they exist inside the ``{
+}`` of the class. The methods of the class all share the instance variables.
```

Comment:
This is an important point "The methods of the class all share the instance variables." Perhaps it should be earlier.

Reply:
Moved.